http/cookie.ts

```diff
@@ -218,3 +218,97 @@ export function deleteCookie(
     ...attributes,
   });
 }
+
+/**
+ * @param value
+ * @returns
+ */
+function parseSetCookie(value: string): Cookie {
+  const attrs = value
+    .split(";")
+    .map((attr) =>
+      attr
+        .trim()
+        .split("=")
+        .map((keyOrValue) => keyOrValue.trim())
+    );
+  const cookie: Cookie = {
+    name: attrs[0][0],
+    value: attrs[0][1],
+  };
+
+  for (const [key, value] of attrs.slice(1)) {
+    switch (key.toLocaleLowerCase()) {
+      case "expires":
+        cookie.expires = new Date(value);
+        break;
+      case "max-age":
+        cookie.maxAge = Number(value);
+        assert(
+          cookie.maxAge >= 0,
+          "Max-Age must be an integer superior or equal to 0",
```

Comment:
similar to the below comment, how about just showing warning instead of throwing?

Reply:
@iuioiua How about making this `console.warn` instead of throwing?

Reply:
Oh, missed this. Fixed now.